web_client/models/WidgetModel.js

```diff
@@ -32,6 +25,13 @@ var WidgetModel = Backbone.Model.extend({
     },
 
     /**
+     * Sets initial model attributes with normalization.
+     */
+    initialize: function (model) {
+        this.set(_.defaults(model || {}, this.defaults));
```

Comment:
This is a perfectly valid point, but outside the scope of this PR which is just trying to adjust to the new eslint rules and structural changes in Girder.